nest/src/users/users.service.ts

```diff
@@ -29,6 +30,7 @@ export class UsersService {
       const User = await this.prisma.user.create({
         data: {
           name: createUserDto.name,
+          intraId: -1, // Needs to be adjusted
```

Comment:
👍🏻

Reply:
what should I put as this point ? can this really happen ? I mean I know that if the user skip the auth via the 42api this will happen : but in our app flow this won't happen isn't ?

Reply:
As the user will be created in the OAuth2 process only, the whole function can go away.
I just didn't want to break the existing flow 👍

Reply:
true got it thanks !